auth_saml/controllers/main.py

```diff
@@ -270,3 +271,20 @@ def saml_metadata(self, req, **kw):
                 ),
                 [("Content-Type", "text/xml")],
             )
+
+
+class Session(WebSession):
+    @http.route("/web/session/logout", type="http", auth="none")
+    def logout(self, redirect="/web/login"):
```

Comment:
I'm wondering if I should not just say 
```suggestion
    def logout(self, redirect="/web/login?disable_autoredirect="):
```

and simply  `return super().logout(redirect)`

Reply:
hello @gurneyalex , reading the doc https://github.com/OCA/server-auth/blob/15.0/auth_saml/README.rst#configuration I think we could try to test it and have an answer

Reply:
I've tried this variant, seems we have the same expected result